### PR TITLE
Added hotels display

### LINKED ISSUE #45: Fix Main Page UI, Fix reservations (effort 3)
Main Page UI:
- [x] 1. Show hotels for a selected city or using Search Bar
- [x] 2. Remove 'All Hotels' just show city name in the left top corner
- [x] 3. Search bar should be a form and everything should be required
- [x] 4. 'Anywhere' City in Seach Bar should be set by deault
- [x] 5. after clicking Search we navigte to view with all hotels with available rooms. Add UI for it (check if Irek havne t done it). Then when we click hotel, open page with rooms for this hotel and SHOW room price for selected period of time (eg. 6days, room price $150, then show 6*150)

### PR DESCRIPTION
**Pull request by GoraJakub**

### CODE REVIEW

File: src/components/HotelCard.jsx [hunk 1]
```diff
@@ -52,12 +52,15 @@ export function HotelCard({ hotel }) {
               readOnly
             />
           </Grid>
-          <p>({hotel.localization.city})</p>
+          <p>{hotel.localization.city}</p>
```
Comment (pdybowski) [inline src/components/HotelCard.jsx]: leave it inside parentheses

File: src/components/RoomCard.jsx [hunk 2]
```diff
@@ -65,7 +50,7 @@ export function RoomCard(room, isLogged) {
         <Grid container direction="column" item xs={2}>
           <span>
             <EuroIcon className={classes.icon} />
-            Price: {room.price}
+            Price for {days} {days > 1 ? 'days' : 'day'}: {room.price * days}
```
Comment (pdybowski) [inline src/components/RoomCard.jsx]: just 
```Price: ${room.price*days}```

File: src/components/views/mainPageView/Hotels.jsx [hunk 3]
```diff
@@ -21,27 +23,36 @@ export const Hotels = ({ match, location }) => {
   let data = location.state ? location.state : { city: match.params.data }
   data = !data.startDate && data.city === 'Anywhere' ? '' : data
 
+  const calulateDays = () => {
```
Comment (pdybowski) [inline src/components/views/mainPageView/Hotels.jsx]: spelling

File: src/components/views/mainPageView/Hotels.jsx [hunk 4]
```diff
@@ -21,27 +23,36 @@ export const Hotels = ({ match, location }) => {
...
+          {hotels.hotels.map((hotel) => {
+            const days = calulateDays()
+            return <HotelCard hotel={hotel} days={days} />
+          })}
+        </>
       ) : (
         <h1>No hotels found in {data.city}</h1>
       )}
-      {/* There must be loop with HotelCard */}
+      {console.log(hotels)}
```
Comment (pdybowski) [inline src/components/views/mainPageView/Hotels.jsx]: remove console.log

File: src/components/views/mainPageView/SearchBar.jsx [hunk 5]
```diff
@@ -8,6 +7,8 @@ import InputLabel from '@material-ui/core/InputLabel'
 import FormControl from '@material-ui/core/FormControl'
 import Select from '@material-ui/core/Select'
 import Button from '@material-ui/core/Button'
+import { Redirect } from 'react-router-dom'
+import '../../../content/css/searchBar.css'
```
Comment (pdybowski) [inline src/components/views/mainPageView/SearchBar.jsx]: /core/vendor.js

File: src/components/views/mainPageView/SearchBar.jsx [hunk 6]
```diff
@@ -36,85 +38,88 @@ const SearchBar = ({ onSearchSubmit }) => {
     children,
   }
 
+  const handleOnSubmit = (e) => {
```
Comment (pdybowski) [inline src/components/views/mainPageView/SearchBar.jsx]: just submit()

**Review by pdybowski — COMMENTED** [review]
small fixes

File: src/routes/general.js [hunk 7]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { Route } from 'react-router-dom'
 import App from '../App'
-import { Register } from '../components'
+import { HotelMoreDetails, Register } from '../components'
 import { Hotels } from '../components/views/mainPageView/Hotels'
 import { NoMatch } from '../components/NoMatch'
```
Comment (pdybowski) [inline src/routes/general.js]: Can you import NoMatch just from Components? Same with Hotels

Comment (GoraJakub) [reply src/routes/general.js]: Done

File: src/components/views/mainPageView/index.jsx [hunk 8]
```diff
@@ -61,3 +61,5 @@ export const MainPageView = () => {
     </div>
   )
 }
+
+export * from './Hotels'
```
Comment (pdybowski) [inline src/components/views/mainPageView/index.jsx:65]: Nah, remove it from here

File: src/routes/general.js [hunk 9]
```diff
@@ -1,9 +1,7 @@
 import React from 'react'
 import { Route } from 'react-router-dom'
 import App from '../App'
-import { HotelMoreDetails, Register } from '../components'
-import { Hotels } from '../components/views/mainPageView/Hotels'
```
Comment (pdybowski) [inline src/routes/general.js:5]: Give it back, i will think later how to fix it

**Pull request merged**